content/en/docs/rate-limits.md

```diff
@@ -3,131 +3,140 @@ title: Rate Limits
 slug: rate-limits
 top_graphic: 1
 date: 2018-01-04
-lastmod: 2020-03-05
+lastmod: 2020-12-11
```

Comment:
One last update pre-merge?

Reply:
I'm not understanding. Did I not modify the lastmod date correctly?

I think maybe your joking with me about my willingness to make another modification. If so, yeah, I'm willing to modify. 😊